### ISSUE TITLE
400 Bad Request when removing someone

### ISSUE THREAD

**Issue by chauffer** [jgeboski/bitlbee-steam#98]
```
 @simone | remove person                                                                  
     <-- | person (steamid@steam) has quit (Removed)                            
   @root | Buddy `steamid' (nick person) removed from contact list              
   @root | steam - Error: HTTP: 400 Bad Request                                               
   @root | steam - Signing off..                                      
```
I'm running the latest version of bitlbee.

**Comment by jgeboski** [comment]
They are really breaking a lot lately, lovely.

**Comment by CaptainYeowang** [comment]
This also seems to happen when I accept a new buddy into my list.
I have to remove/add people through the website or the actual Steam client.

Sorry, no logs, just thought I'd chime in and add this.

**Comment by jgeboski** [comment]
Resolved as of 2c39ddb.
